### DISCUSSION TITLE
Mac os Apple m1 how to install the plugin and does it support OBS 28 version?

### DISCUSSION THREAD

**Discussion by AntonSokolov1985** [WarmUpTill/SceneSwitcher#508] [General]
how to install the distribution kit on the beta version? it does not appear in plugins when I install version 1.17

**Comment by WarmUpTill** [comment]
Version 1.17 is not compatible with OBS 28.
A version of the plugin for OBS 28 is available here:
https://github.com/WarmUpTill/SceneSwitcher/actions/runs/2914689999

I will release this new version once OBS 28 is officially released.

**Comment by AntonSokolov1985** [comment]
thank you, I saw it, downloaded it, everything worked. This plugin should be a base so that you don’t deal with such a collective farm, it’s strange that the OBS developers don’t understand this

**Comment by AntonSokolov1985** [comment]
release of version 28 took place

**Reply by WarmUpTill** [reply]
Thanks for the hint - working on a release right now.
(Not sure if I will finish it today, but should be ready tomorrow)

**Comment by AntonSokolov1985** [comment]
Очень жду !  спасибо большое !

чт, 1 сент. 2022 г. в 23:40, WarmUpTill ***@***.***>:
